examples/transforms/global_vs_local_translation.rs

```diff
@@ -0,0 +1,141 @@
+use bevy::prelude::*;
+
+// Define a struct to store basic state information.
+struct EntityState {
+    spawn_location: Vec3,
+    stopped: bool,
+}
+
+// Define a marker for entities that should stop at a distance in respect to their global transform.
+struct GlobalStop;
+// Define a marker for entities that should stop at a distance in respect to their local transform.
+struct LocalStop;
+
+// Define the maximum distance an entity should be able to move away from its spawn.
+const MAX_DISTANCE: f32 = 5.0;
+
+// Ease up creation of EntityState structs.
```

Comment:
I think this is probably overkill for this example, and would just use struct literal syntax to initialize these objects.